src/setting.rs

```diff
@@ -133,6 +133,42 @@ impl SettingData {
     }
 }
 
+/// Payload of a `DEVSETTING.DAT` file (32 bytes).
+#[binrw]
+#[derive(Debug, PartialEq)]
+#[brw(little)]
+pub struct DevSetting {
+    /// Unknown field.
+    unknown1: [u8; 9],
+    /// "Type of the overview Waveform" setting.
+    pub overview_waveform_type: OverviewWaveformType,
+    /// "Waveform color" setting.
+    pub waveform_color: WaveformColor,
+    /// Unknown field.
+    unknown2: u8,
+    /// "Key display format" setting.
+    pub key_display_format: KeyDisplayFormat,
+    /// "Waveform Current Position" setting.
+    pub waveform_current_position: WaveformCurrentPosition,
+    /// Unknown field.
+    #[br(assert(unknown3 == [0x00; 18]))]
+    unknown3: [u8; 18],
+}
+
+impl Default for DevSetting {
+    fn default() -> Self {
+        Self {
+            unknown1: [0x78, 0x56, 0x34, 0x12, 0x01, 0x00, 0x00, 0x00, 0x01],
```

Comment:
Tbh I have no idea what these bytes mean. These seem to be the default values, but I guess there are connected to some settings I didn't discover yet.

Reply:
Interestingly, these bytes are largely identical to the bytes in `DJMMYSETTING.DAT`...

https://github.com/Holzhaus/rekordcrate/blob/cd1edd75282030b9894eb6347247a4940ab19f14/src/setting.rs#L178

... and `MYSETTING.DAT`...

https://github.com/Holzhaus/rekordcrate/blob/cd1edd75282030b9894eb6347247a4940ab19f14/src/setting.rs#L267

... but tjey are missing in `MYSETTING2.DAT`.

Reply:
> Tbh I have no idea what these bytes mean. These seem to be the default values, but I guess there are connected to some settings I didn't discover yet.

Yeah, even just that as a comment sufficies to document that there is no knowledge we left out that make up these bytes. 

> Interestingly, these bytes are largely identical to the bytes in DJMMYSETTING.DAT...

Mhmm thats indeed interesting. Maybe its some required setting of older hardware/rekordbox versions that they removed in the current version? At least thats my guess assuming `MYSETTING2` got introduced after `DJMMYSETTING` and `MYSETTING`.

Reply:
Comparing `MYSETTING` and `DJMMYSETTING`, seems to me like the first byte is some magic signature while `MYSETTING` has a `u32` and `DJMMYSETTING` has two `u32`s after the signature.

Reply:
Possible, but we don't really know. This can be determined later on.

> Yeah, even just that as a comment sufficies to document that there is no knowledge we left out that make up these bytes.

I think by naming the field `unknown` and the assertion added in https://github.com/Holzhaus/rekordcrate/pull/62/commits/e370cb93854ba5d04e7e6868702435244f45884b it's clear now that these fields always had this specific value in all samples that we looked at.